drivers/serial/uart_stm32.c

```diff
@@ -233,8 +233,7 @@ static inline uint32_t uart_stm32_get_parity(const struct device *dev)
 	return LL_USART_GetParity(config->usart);
 }
 
-static inline void uart_stm32_set_stopbits(const struct device *dev,
-					   uint32_t stopbits)
+static inline void uart_stm32_set_stopbits(const struct device *dev, uint32_t stopbits)
```

Comment:
Don't mix formatting changes with functional changes in same commit, this highly complexify the review process.